packages/expo-local-authentication/src/__tests__/LocalAuthentication-test.ts

```diff
@@ -0,0 +1,34 @@
+import { mockPlatformAndroid, mockPlatformIOS, unmockAllProperties } from 'jest-expo';
+
+import ExpoLocalAuthentication from '../ExpoLocalAuthentication';
+import * as LocalAuthentication from '../LocalAuthentication';
+
+afterEach(unmockAllProperties);
+
+beforeEach(() => {
+  ExpoLocalAuthentication.authenticateAsync.mockImplementation(async () => ({}));
+});
+it(`uses message on iOS`, async () => {
+  mockPlatformIOS();
+
+  const message = '<DEBUG_MESSAGE>';
+  await LocalAuthentication.authenticateAsync(message);
+
+  expect(ExpoLocalAuthentication.authenticateAsync).toHaveBeenCalledWith(message);
```

Comment:
minor: using `toHaveBeenLastCalledWith` looks at just the last call rather than checking if there was any call with the specified arguments. This way, if the mocking/unmocking between tests were to fail, we'd still have some isolation between tests and make it more likely to catch bugs.